pkg/apis/pipeline/v1beta1/pipelinerun_types.go

```diff
@@ -408,10 +427,12 @@ type PipelineRunStatusFields struct {
 	// +optional
 	CompletionTime *metav1.Time `json:"completionTime,omitempty"`
 
+	// Deprecated - use ChildReferences instead.
```

Comment:
optional comment: I wonder if it's possible to add an annotation that will allow these fields to display as deprecated in code editors? I did a quick search and see `// +k8s:prerelease-lifecycle-gen:deprecated=1.20` for some stuff in our k8s dependencies so it's probably possible